apps/webservice/src/env.ts

```diff
@@ -23,6 +23,7 @@ export const env = createEnv({
     GITHUB_BOT_APP_ID: z.string().optional(),
     GITHUB_BOT_PRIVATE_KEY: z.string().optional(),
     BASE_URL: z.string(),
+    NEXT_RUNTIME: z.enum(["edge", "nodejs"]).default("nodejs"),
```

Comment:
this will never be edge, we should remove this and hardcode it

Reply:
Sounds good 👍